package.json

```diff
@@ -10,8 +10,8 @@
   },
   "dependencies": {
     "next": "^14.2.5",
-    "react": "^18",
-    "react-dom": "^18"
+    "react": "^18.3.1",
```

Comment:
I don't think we should update the react version if its currently working for everyone